web_sdk/BUILD.gn

```diff
@@ -77,7 +80,10 @@ prebuilt_dart_action("web_engine_sources") {
   input_dir = rebase_path("//flutter/lib/web_ui/lib/src/")
   pool = "//flutter/build/dart:dart_pool"
 
-  outputs = [ "$target_gen_dir/$target_name.stamp" ]
+  outputs = [
+    "$target_gen_dir/$target_name.stamp",
+    "$root_out_dir/flutter_web_sdk/lib/_engine/",
```

Comment:
I think you can just use the `$output_dir` variable here, right?

Reply:
Using $output_dir is returning the following error:

```
ERROR at //flutter/web_sdk/BUILD.gn:57:5: File is not inside output directory.
    "$output_dir",
    ^------------
The given file should be in the output directory. Normally you would specify
"$target_out_dir/foo" or "$target_gen_dir/foo". I interpreted this as
"/usr/local/google/home/godofredoc/flutter/engine/src/out/host_debug/flutter_web_sdk/lib/ui/".
```